src/metaImage.cxx

```diff
@@ -17,6 +17,7 @@
 #include <string.h> // for memcpy
 #include <stdlib.h> // for atoi
 #include <math.h>
+#include <algorithm> // std::max
```

Comment:
& std::min. Nitpick, add 'for' (see above)

Reply:
Make sense